test/ctia/test_helpers/graphql.clj

```diff
@@ -1,9 +1,9 @@
 (ns ctia.test-helpers.graphql
-  (:require [clojure.test :refer [is testing]]
+  (:require [clojure
+             [string :as str]
```

Comment:
I know that it was already here, but `str` is a clojure function, would you mind to rename that alias into `string`, it is used only once in that ns: https://github.com/threatgrid/ctia/blob/631468b35718f24ec2521c0ab8ad35cb506392f5/test/ctia/test_helpers/graphql.clj#L62